### PR TITLE
Cache predictions in the new endpoint.

### PR DESCRIPTION
**Pull request by codeviking**
This adds code for serving cached predictions to the new endpoint.

### CODE REVIEW

File: allennlp_demo/bidaf/api.py [hunk 1]
```diff
@@ -23,9 +27,29 @@
     def info() -> str:
         return jsonify({ **asdict(model), "allennlp": VERSION })
 
+    @lru_cache(maxsize=128)
```
Comment (epwalsh) [inline allennlp_demo/bidaf/api.py:30]: I think we could make this an order of magnitude higher at least, right?

Comment (codeviking) [reply allennlp_demo/bidaf/api.py:30]: Maybe. I was trying to change as little as possible -- so I maintained the existing value.

@schmmd @epwalsh If either of you tell me a magic # to put here instead I'm happy to add it.

Comment (schmmd) [reply allennlp_demo/bidaf/api.py:30]: 1024 is my suggestion

Comment (codeviking) [inline allennlp_demo/bidaf/api.py:30]:
```suggestion
    @lru_cache(maxsize=1024)
```

Comment (codeviking) [reply allennlp_demo/bidaf/api.py]: Done.

**Pull request merged**